osu.Game.Rulesets.Osu/Scoring/OsuScoreProcessor.cs

```diff
@@ -23,11 +23,19 @@ public OsuScoreProcessor(RulesetContainer<OsuHitObject, OsuJudgement> rulesetCon
         {
         }
 
+        
+        float beatmapHp;
+
+        protected override void ComputeTargets(Game.Beatmaps.Beatmap<OsuHitObject> beatmap)
+        {
+            beatmapHp = beatmap.BeatmapInfo.Difficulty.DrainRate;
+        }
+
         protected override void Reset()
         {
             base.Reset();
 
-            Health.Value = 1;
+            Health.Value = 0.5;

```

Comment:
Why did this change happen?

Reply:
To show me how much hp was gained while testing.

Oops.